tests/core/test_wire.py

```diff
@@ -902,3 +907,31 @@ def test_french_accents_search(client, app):
     assert 1 == len(resp.json["_items"])
     resp = client.get("/wire/search?q=electión")
     assert 1 == len(resp.json["_items"])
+
+
+def test_navigation_for_public_users(client, app, setup_products):
+    user = app.data.find_one("users", req=None, _id=PUBLIC_USER_ID)
+    assert user
+
+    company = app.data.find_one("companies", req=None, _id=COMPANY_1_ID)
+    assert company
+
+    # add products to user
+    app.data.update(
+        "users", PUBLIC_USER_ID, {"products": [{"section": "wire", "_id": 10}, {"section": "wire", "_id": 11}]}, user
+    )
+
+    # and remove those from company
+    app.data.update("companies", COMPANY_1_ID, {"products": None}, company)
```

Comment:
How can a user have access to a product when the parent company does not? Shouldn't this be a requirement?

Reply:
in general that is not the case, I've just simplified the setup for the test

Reply:
updated to test to avoid any confusion